Tasks/ArchiveFiles/task.json

```diff
@@ -70,7 +70,7 @@
             "groupName": "archive",
             "visibleRule": "archiveType = tar",
             "options": {
-                "default": "gz",
```

Comment:
Good question.  I confirmed that this change only affects the UI: the drop-down displays an initial value instead of being blank.  But the behavior doesn't change.